tests/components/component.jsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react'
+import {Component} from 'react'
```

Comment:
You are. All the JSX components you write are converted to `React.createElement('div', ...)` Babel transpiles this but it doesn't import it.